<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterForm</class>
 <widget class="QDialog" name="RegisterForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="tomeoLogo">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>261</width>
     <height>251</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="logo.qrc">:/Assets/Untit.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="privacy">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: transparent;
    border: none;
    color: blue;
    font-size: 13px;
    padding: 5px;
    font-weight: bold;
}

QPushButton:hover {
    color: darkblue;
}
</string>
   </property>
   <property name="text">
    <string>Privacy</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginBtn">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>0</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: transparent;
    border: none;
    color: blue;
    font-size: 13px;
    padding: 5px;
    font-weight: bold;
}

QPushButton:hover {
    color: darkblue;
}
</string>
   </property>
   <property name="text">
    <string>Log in</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="usernameEntry1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>310</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Enter Username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passentry1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>360</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Enter Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="confirm_pass1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>410</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Confirm Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ConfirmRegBtn">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>470</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #414bb2;
    color: white;
    border-radius: 10px;
	font: 11pt &quot;Ebrima&quot;;
}
</string>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="logo.qrc"/>
 </resources>
 <connections/>
</ui>
